rules/windows/powershell/powershell_malicious_commandlets.yml

```diff
@@ -10,110 +10,114 @@ tags:
     - attack.t1086  #an old one
 author: Sean Metcalf (source), Florian Roth (rule)
```

Comment:
Hello @bczyz1 !  I believe you can put there your name and oscd.community with `(update)` (:

Reply:
Hey @yugoslavskiy, thanks, wasn't sure about that and decided to play it safe 😃 authors appended in e90f91b89ee905897d764de581a1e7ae000390ce, cheers!